resources/utils/index.ts

```diff
@@ -58,6 +58,32 @@ function getErrorCode (e: Error) {
   return e.code
 }
 
+function accountSort (accounts:Record<string, Account>, a:string, b:string) {
+  try {
+    let [aBlockStr, aLocalStr] = accounts[a].created.split(':')
+    let [bBlockStr, bLocalStr] = accounts[b].created.split(':')
+
+    const aLocal = parseInt(aLocalStr)
+    const bLocal = parseInt(bLocalStr)
+
+    if (aBlockStr === 'new' && bBlockStr !== 'new') return -1
+    if (bBlockStr !== 'new' && aBlockStr === 'new') return 1
+    if (aBlockStr === 'new' && bBlockStr === 'new') return aLocal >= bLocal ? -1: 1
```

Comment:
can be simplified to `if (aBlockStr === 'new' && bBlockStr === 'new') return bLocal - aLocal`